libiconv-wasm32.patch

```diff
@@ -1,6 +1,6 @@
-diff -ruN libiconv-1.15/build-aux/config.sub libiconv-1.15-patched/build-aux/config.sub
---- libiconv-1.15/build-aux/config.sub	2017-01-02 09:02:47.000000000 +1000
-+++ libiconv-1.15-patched/build-aux/config.sub	2017-11-22 08:38:36.995036311 +1000
```

Comment:
Yea it'd be nice if the patch directory names could have stayed the same. i.e. `libiconv-1.15` and `libiconv-1.15-patched`.

Reply:
I'll fix.